### DISCUSSION TITLE
Question pertaining to Normalization for Multiome Integration

### DISCUSSION THREAD

**Discussion by sylestiel** [stuart-lab/signac#1521] [Q&A] [answered]
@timoast 

When INTEGRATING Multiome (RNA +ATAC) datasets do we need to normalize the RNA data and the ATAC data for **each** of the samples  **individually** **and after integration**?  Kindly clarify.

Thanks!

**Comment by timoast** [comment, marked as answer]
You can normalize gene expression data before integration and run  `RunTFIDF()` after integration

**Reply by shahrozeabbas** [reply]
Hi, just wanted to clarify this further. The way I've been doing it is to normalize RNA and ATAC per dataset, identify top features for both modalities, merge, run pca/lsi, then harmony to remove batch effects. 

Wouldn't you need to normalize ATAC with `RunTFIDF()` to integrate datasets? Here, I assume integration refers to correcting for batch effects.

**Reply by timoast** [reply]
If you're integrating the ATAC assay then you would need an LSI dimension reduction, and so would need to run TF-IDF beforehand. For multiome you could use the RNA assay to integrate the datasets, depends what you're aiming to do. However TF-IDF should be re-run after adding or removing cells to the dataset since that will change the IDF values for each peak.

**Reply by shahrozeabbas** [reply]
Okay got it, thanks!
